source/2019-04-05-the-ember-times-issue-92.md

```diff
@@ -12,10 +12,16 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [3 Reasons To Volunteer At a Tech Conference 💪🏼](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8)
```

Comment:
Love the 💪

Reply:
Change to `3 Reasons to Volunteer at a Tech Conference 💪🏼`, see https://titlecase.com/. I will also update CONTRIBUTING, this is a better link than the one we have.

Reply:
```suggestion
## [3 Reasons to Volunteer at a Tech Conference 💪🏼](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8)
```